src/site/pages/guidelines.md

```diff
@@ -302,6 +302,8 @@ States of a component should be included as a nested element. This includes hove
 }
 ```
 
+Classes that duplicate a state should be named with the prefix `pf-is-`. For example, a class `pf-is-active` can be used to apply the `:active` styling. 
```

Comment:
ok, good. So if the class doesn't duplicate a state then we can use `-has-`, right? as in `pf-has-gutters`